src/main/java/command/UserInput.java

```diff
@@ -0,0 +1,130 @@
+package command;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.util.InputMismatchException;
+import java.util.List;
+
+public class UserInput {
+    public static void parseInput(String input, Scanner in, List<Task> taskList) {
+
+        int taskID;
+
+        switch (input.toLowerCase()) {
+
+        case "list":
+            printLists(taskList);
+            break;
+
+        case "bye":
+            break;
+
+        case "mark":
+            taskID = in.nextInt();
+            executeMark(taskList, taskID);
+            break;
+
+        case "unmark":
+            try {
+                taskID = in.nextInt();
+                executeUnmark(taskList, taskID);
+            } catch (InputMismatchException e) {
+                System.out.println("A number please");
+            }
+            break;
+
+        case "todo":
+            input = in.nextLine();
+            parseToDo(input, taskList);
+            break;
+
+        case "deadline":
+            input = in.nextLine();
+            parseDeadline(input, taskList);
+            break;
+
+        case "event":
```

Comment:
avoid using magic literals. you can store them in static final variables 
https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers